packages/taquito-local-forging/package.json

```diff
@@ -73,6 +73,7 @@
   },
   "devDependencies": {
     "@taquito/rpc": "^15.0.1",
+    "@taquito/core": "^15.0.0",
```

Comment:
Saw your comments in a channel the other day agree that we should update taquito/core to ^15.0.1 to align with other packages.